core/src/main/kotlin/Formats/HtmlFormatService.kt

```diff
@@ -45,22 +49,20 @@ open class HtmlFormatService @Inject constructor(@Named("folders") locationServi
         to.appendln("$text<br/>")
     }
 
+    override fun appendList(to: StringBuilder, body: () -> Unit) {
+        body()
+    }
+
     override fun appendAnchor(to: StringBuilder, anchor: String) {
         to.appendln("<a name=\"${anchor.htmlEscape()}\"></a>")
     }
 
-    override fun appendTable(to: StringBuilder, body: () -> Unit) {
+    override fun appendTable(to: StringBuilder, columnCount: Int, body: () -> Unit) {
         to.appendln("<table>")
         body()
         to.appendln("</table>")
     }
 
-    override fun appendTableHeader(to: StringBuilder, body: () -> Unit) {
```

Comment:
It wasn't used and to add it causes a problem for Markdown I didn't want to solve unless this is every brought back to be used.  Then ONLY if it is really needed, do the hard work to solve the problem.